### PR TITLE
Made some changes 😎

### PR DESCRIPTION
**Pull request by IGLADI**
Reverts the commit that created a direct exchange for RabbitMQ. (we now use the default exchange again)

Automatically import ndjson to Kibana + remove volume.

Auto update Elasticsearch indice settings, setting tiemstamp to be a real timestamp (someone deleted my previous code for this...)

Remove http/ping heartbeat (client said he didn't needed it)

(Fix last value shown on graph.)

Remove setup container on first down

Remove deprecated docker compose version.

Empty queue on startup for increased speed and remove unrealistic data on graph (100% uptime when we were down)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,27 +61,18 @@ If you'd like to add a service to monitor, please follow these steps:
...
+nano ./heartbeat_rabbitmq.csv
 ```
 
 **Notes**
 
--   We check for new yaml files every 5 seconds, consider it may take up to 10 seconds (with both the dashboard and the service set to reload every second) before showing up.
--   We also update the consumer with the csv file every minute, so it may take up to 1.5 minutes before the service is fully operational.
--   While a yml file isn't configured properly, we recommend to keep the `.unconfirmed` extension.
--   If you temporarily don't want to monitor a service you can set `enabled: false` in the yaml file.
+-   We check for new csv file every x time, consider it may take up to 30 seconds (with both the dashboard and the service set to reload every second) before showing up.
```
Comment (luluwaza) [inline README.md]: quantify "x"

**Comment by JDW-ehb** [comment]
Looks great, automatisation of the export of index patterns and saved object is a nice automatisation and also will fasten future processes.

**Comment by IGLADI** [comment]
> Looks great, automatisation of the index patterns and saved object is a nice automatisation and also will fasten future processes.

Thx bro, ik you wish you had me in your team 😎

File: src/setup/export_setup.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+print("host: ", host)
+print("url: ", url)
+
+session = requests.Session()
+
+url2 = f"{url}/status"
+
+print(f"Waiting for kibana API to be up at {url2}...")
+while True:
+    try:
```
Comment (luluwaza) [inline src/setup/export_setup.py:23]: May want to add some comments for clarity

Comment (IGLADI) [reply src/setup/export_setup.py:23]: What's not clear? The print already tells what's happening

File: README.md (hunk 1 above)
Comment (IGLADI) [reply README.md]: [Done](https://github.com/Integration-Project-Team-1/Monitoring/pull/2/commits/a0030b19534ecee670c0474a6c4116464682cd34)

**Review by luluwaza — APPROVED** [review]
Looks good overal, added comments where extra explanation / modifications would be nice but aren't necessary

File: src/setup/export_setup.py (hunk 2 above)
Comment (luluwaza) [reply src/setup/export_setup.py:23]: Sorry wrong line, ment line 32-38 which aren't clear on a first read-through

Comment (IGLADI) [reply src/setup/export_setup.py:23]: [Done](https://github.com/Integration-Project-Team-1/Monitoring/pull/2/commits/7293a5c83fba3eabc0a88bd52a2f4c8dc57d218f)

**Pull request merged**